apps/console/src/features/applications/api/application.ts

```diff
@@ -984,3 +984,83 @@ export const getRequestPathAuthenticators = (): Promise<any> => {
                 error.config);
         });
 };
+
+/**
+ * Function to add/update My Account portal status.
+ *
+ * @param {boolean} status - My Account portal status.
```

Comment:
Check for ESLint errors in the doc comments. Now we use TSDoc. https://tsdoc.org/

Reply:
Addressed with commit 95b6b5f048609693f173d31db083730da8286d99